package.json

```diff
@@ -23,6 +23,7 @@
   },
   "files": [
     "lib",
+    "manifest.json",
```

Comment:
This makes sure that the manifest.json file is included in the published version of the project